.github/workflows/update.yaml

```diff
@@ -34,6 +34,9 @@ jobs:
 
       - name: Run webshots script
         run: xvfb-run python tools/make_webshots.py
+        env:
+          DANDI_USERNAME: ${{ secrets.DANDI_USERNAME }}
+          DANDI_PASSWORD: ${{ secrets.DANDI_PASSWORD }}
```

Comment:
iirc it includes `$` - I wonder if there is necessity to surround the value in `'` ?

Reply:
This isn't being parsed by the shell, so that shouldn't be necessary.